backend/manager/modules/bll/src/main/java/org/ovirt/engine/core/bll/exportimport/ImportVmCommandBase.java

```diff
@@ -508,13 +512,15 @@ protected StorageDomain getStorageDomain(Guid domainId) {
     @Override
     protected void executeVmCommand() {
         try {
+            handleVmLease();
```

Comment:
this happens before adding the vm to the database so there's no allocation of mac addresses and we can put it outside of the try-block, right?